### PR TITLE
upgrade cody music and share option cleanup

### PR DESCRIPTION
**Pull request by xavluiz**
- this updates the cody-music npm based on the latest version and updates the share text and available options. x.com is still using twitter.com/intent/post (https://developer.x.com/en/docs/twitter-for-websites/tweet-button/guides/web-intent)

Screenshot 2024-06-04 at 5 20 54 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,234 +1,234 @@
...
+		"compile:extension": "tsc -p ./",
+		"compile:views": "webpack --mode development",
+		"watch:extension": "tsc -watch -p ./",
+		"watch:views": "webpack --watch --mode development",
+		"vscode:prepublish": "webpack --mode production",
+		"test-compile": "tsc -p ./"
+	},
+	"contributes": {
+		"commands": [
+			{
```
Comment (bjacobson26) [inline package.json:60]: the whitespace is a bit much

File: src/social/SocialShareManager.ts [hunk 2]
```diff
@@ -20,218 +20,174 @@ let spotifyLinkUrl: string = "";
...
+		return SocialShareManager.instance;
+	}
+
+	shareIt(sharer: string, options: any) {
+		const shareUrl = this.getShareUrl(sharer, options);
+		launchWebUrl(shareUrl);
+	}
+
+	getShareUrl(sharer: string, options: any) {
+		console.log('options', options)
```
Comment (bjacobson26) [inline src/social/SocialShareManager.ts:43]: remove console log

**Review by bjacobson26 — COMMENTED** [review]
whitespace looks off in this project - add eslint?

File: package.json (hunk 1 above)
Comment (xavluiz) [reply package.json]: yeah, not sure what happened. used prettier to reformat

**Review by bjacobson26 — APPROVED** [review]

**Pull request merged**